avocado/utils/pmem.py

```diff
@@ -52,7 +53,7 @@ def __init__(self, ndctl='ndctl', daxctl='daxctl'):
         :param daxctl: path to daxctl binary, defaults to ndctl
         """
         for lib_bin in [ndctl, daxctl]:
-            if process.system('which %s' % lib_bin, shell=True, ignore_status=True):
+            if not path.find_command(lib_bin):
                 raise PMemException("Cannot use library without "
                                     "proper binary %s" % lib_bin)
         self.ndctl = ndctl
```

Comment:
`find_command` looks up in funky locations... passing the check above does not mean the command is in `$PATH`.

Reply:
That's a good point, which makes me think of the following options:
1) have a way that `find_command` will stick to `$PATH`
2) set the `ndctl` and `daxctl` attributes to the paths eventually found by `find_command`
Preferences? Other suggestions?

Reply:
I'd go with n. 2 :)